firefox-ios/firefox-ios-tests/Tests/XCUITests/BrowsingPDFTests.swift

```diff
@@ -115,7 +115,6 @@ class BrowsingPDFTests: BaseTestCase {
         navigator.openURL(PDF_website["url"]!)
         waitUntilPageLoad()
         navigator.performAction(Action.BookmarkThreeDots)
-        waitUntilPageLoad()
```

Comment:
The page isn't reloaded on iOS 18 for sure. Removing this line seems to work in older iOS versions.